src/FeatureTable.js

```diff
@@ -1,18 +1,13 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { Button } from 'react-bootstrap'
+import ProductRow from './ProductRow'
 
 export default class FeatureTable extends Component {
   render () {
     let rows = []
+
     this.props.products.forEach((productRow) => {
-      rows.push(
-        <tr className='product-row'>
-          <td><Button>edit</Button></td>
-          <td>{productRow}</td>
-          <td />
-        </tr>
-      )
+      rows.push(<ProductRow productRow={productRow} />)
```

Comment:
prefer `map()` over `forEach()` whenever you can. For example, this could have been written:
```
const rows = this.props.products.map((productRow) => {
  return (<ProductRow productRow={productRow} />)
})
```